### DISCUSSION TITLE
Suggestion: apt repository

### DISCUSSION THREAD

**Discussion by sbrl** [knqyf263/pet#285] [Ideas]
I've noticed that .deb packages are provided for this program already.

An apt repository would allow people to update Debian-based systems automatically.

I found [aptirepo](https://github.com/opinsys/aptirepo), which appears to reatly simplify the process of setting up a simple apt repository. It would appear that all that's required is the hosting of static files from a directory somehow.

**Comment by RamiAwar** [comment]
Thanks for the suggestion @sbrl !
